src/hooks/useAppState/selectors.js

```diff
@@ -0,0 +1,3 @@
+export const selectItems = (state) => state.items[state.index] ?? [];
+export const selectLists = (state) => state.lists ?? [];
+export const selectListId = (state) => state.lists[state.index]?.id ?? null;
```

Comment:
These are the new helper methods to get specific things out of the state, the question marks are these:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator